packages/terra-application-navigation/tests/wdio/application-navigation-spec.js

```diff
@@ -122,6 +122,26 @@ Terra.describeViewports('ApplicationNavigation - Large', ['large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('Should close open popup when custom event is dispatched', () => {
+    before(() => browser.url('/#/raw/tests/terra-application-navigation/application-navigation/application-navigation'));
+
+    it('launch popup', () => {
+      browser.click('[data-application-header-utility="true"]');
+      browser.waitForVisible('[data-terra-popup-content="true"]');
+    });
+
+    Terra.it.validatesElement('1. Popup Open');
+
+    it('close popup', () => {
+      // eslint-disable-next-line prefer-arrow-callback
+      browser.execute(function dispatch() {
+        window.dispatchEvent(new Event('terra-application-navigation.dismiss-menu'));
```

Comment:
ie doesn't support event constructors, so we'd need to do something similar to this 
https://github.com/cerner/terra-application/blob/main/tests/wdio/application-base/private/test-overrides-spec.js#L6-L9

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/1255/commits/60dc2ba504e31fd36dbc694fc4b95e9d046848f2#diff-85a7fe8bb38bbc87cbbac08c28ef3370R139